apps/server/src/api-data/sheets/sheets.controller.ts

```diff
@@ -56,6 +57,16 @@ export async function revokeAuthentication(_req: Request, res: Response) {
   }
 }
 
+export async function getWorksheetNamesFromSheet(req: Request, res: Response) {
+  try {
+    const { sheetId } = req.params;
+    const { worksheetOptions } = await getWorksheetOptions(sheetId);
```

Comment:
if this function has a single return, I dont see why we need the overhead of wrapping it in an object? am i missing something?